primefaces/src/main/java/org/primefaces/util/ComponentUtils.java

```diff
@@ -630,8 +630,9 @@ public static void encodeIndexedId(FacesContext context, UIComponent component,
         context.setResponseWriter(writer);
 
         // append index to all id's
+        char separator = UINamingContainer.getSeparatorChar(context);
```

Comment:
Is this a costly operation? If so maybe cache the separator char?

Reply:
I don't think its costly its done all over the code base.  Maybe a question for @tandraschko whether we want to cache it in `Lazy` in PrimeEnvironment?

Reply:
Just asking as we are iterating over a list of items.. 🙂